src/components/AudioPlayer/AudioPlayer.stories.mdx

```diff
@@ -0,0 +1,166 @@
+import { action } from "@storybook/addon-actions";
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs";
+import { withDesign } from "storybook-addon-designs";
+
+import AudioPlayer from "./AudioPlayer";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("AudioPlayer")}
+  component={AudioPlayer}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=36835%3A25338",
+    },
+    jest: ["AudioPlayer.test.tsx"],
+  }}
+  argTypes={{
+    audioType: {},
+    className: { control: false },
+    id: { control: false },
+  }}
+/>
+
+# Audio Player
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.2.0`    |
+| Latest            | `1.2.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [Example Embed Code Snippets](#example-embed-code-snippets)
+- [Errored](#errored)
+
+## Overview
+
+<Description of={AudioPlayer} />
+
+The `AudioPlayer` component is used to embed a streaming audio player from 3rd party services such as `Libsyn`, `SoundCloud`, and `Spotify`
+on a page. The component requires the `embedCode` prop or `filePath` prop. The component will not function properly if you try to set
+both of them.
+
+**This version will not be used for the playback of locally hosted audio files**.
+
+## Component Props
+
+<Canvas withToolbar>
+  <Story
+    name="AudioPlayer"
+    args={{
+      className: undefined,
+      descriptionText:
+        "Audio description lorem ipsum dolor simet. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Sed posuere consectetur est at lobortis.",
+      embedCode: `<iframe title="Libsyn Player" style="border: none" src="//html5-player.libsyn.com/embed/episode/id/18268511/height/90/theme/custom/thumbnail/yes/direction/backward/render-playlist/no/custom-color/87A93A/" height="90" width="100%" scrolling="no"  allowfullscreen webkitallowfullscreen mozallowfullscreen oallowfullscreen msallowfullscreen></iframe>`,
+      headingText: "Audio Title",
+      helperText: "Audio helper text lorem ipsum dolor simet.",
+      id: "audioplayer-id",
+      iframeTitle: "Libsyn Audio",
+      audioType: "libsyn",
+    }}
+  >
+    {(args) => (
+      <AudioPlayer
+        {...args}
+        embedCode={args.embedCode}
+        audioType={args.audioType && args.audioType}
+      />
+    )}
+  </Story>
+</Canvas>
+
+<ArgsTable story="AudioPlayer" />
+
+## Accessibility
+
+The `VideoPlayer` component renders an `iframe` element from `Libsyn`, `SoundCloud`, or `Spotify`.
```

Comment:
Update this parapgrah to the following:

The `VideoPlayer` component renders an iframe element from `Libsyn`, `SoundCloud`, or `Spotify`. The accessibility of the `iframe` element can be improved if a `title` attribute is provided and most 3rd party audio services will include a `title` attribute in the embed code they provide.  If the `title` attribute is omitted from the embed code, a default `title` will be added by the `AudioPlayer` component, but it is recommended to include a custom `title` that is related to the audio file content.

Reply:
Updated. Thank you!